src/patterns/BuilderExample/SportsCarBuilder.java

```diff
@@ -0,0 +1,37 @@
+package patterns.BuilderExample;
+
+import patterns.BuilderExample.entities.Car;
+import patterns.BuilderExample.interfaces.Builder;
+
+public class SportsCarBuilder implements Builder {
+    private Car car;
+
+    public SportsCarBuilder() {
+        this.car = new Car();

```

Comment:
An instance of an object should be created in the method `build`, but not here! It's just a constructor whose purpose is only to create a builder for a sports car but not a car itself.